ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/logging.j2

```diff
@@ -1,19 +1,34 @@
 {# eos - logging #}
 {% if logging is arista.avd.defined %}
 !
-{%     if logging.console is arista.avd.defined %}
+{%     if logging.console is arista.avd.defined('disabled') %}
+no logging console
+{%     elif logging.console is arista.avd.defined %}
 logging console {{ logging.console }}
 {%     endif %}
-{%     if logging.monitor is arista.avd.defined %}
+{%     if logging.monitor is arista.avd.defined('disabled') %}
+no logging monitor
+{%     elif logging.monitor is arista.avd.defined %}
 logging monitor {{ logging.monitor }}
 {%     endif %}
-{%     if logging.buffered.size is arista.avd.defined and logging.buffered.level is arista.avd.defined %}
-logging buffered {{ logging.buffered.size }} {{ logging.buffered.level }}
+{%     if logging.buffered.level is arista.avd.defined('disabled') %}
+no logging buffered
+{%     elif logging.buffered.level is arista.avd.defined %}
+{%         set logging_buffered_cli = "logging buffered "%}
+{%         if logging.buffered.size is arista.avd.defined %}
+{%             set logging_buffered_cli = logging_buffered_cli ~ logging.buffered.size ~ " " %}
```

Comment:
We normally add the space before the word we add, so I propose that you move the `" "` to before `logging.buffered.size`, remove the extra space in line 17 and add a space in line 21. I know it will do the same in this case, but this is just a way to ensure consistent rendering when adding new knobs.
You are making the size optional here - which is great.